package.json

```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "kew": "0.7.0",
     "typ": "0.6.3",
-    "aws-sdk": "2.2.47"
+    "aws-sdk": "2.5.1"
```

Comment:
is there a reason here why we're updating the sdk?

Reply:
Apologies, but this is something I did 2 years ago while implementing something for a former client,  and I don't honestly recall the context here.  If this PR is useful to others I can try and setup something to test it with the previous aws-sdk and see what happens, but I cannot promise when I will have time to do so.